package.json

```diff
@@ -40,16 +39,38 @@
     "url": "https://github.com/fernando-mc/serverless-finch/issues"
   },
   "homepage": "https://github.com/fernando-mc/serverless-finch#readme",
+  "mocha": {
+    "require": [
+      "@serverless/test/setup/patch",
+      "@serverless/test/setup/log",
+      "@serverless/test/setup/mock-homedir",
+      "@serverless/test/setup/mock-cwd",
+      "@serverless/test/setup/restore-env"
+    ],
+    "timeout": 360000
+  },
   "dependencies": {
-    "enquirer": "^2.3.6",
+    "@serverless/utils": "^6.0.2",
     "is_js": "^0.9.0",
     "mime": "^3.0.0",
     "minimatch": "^3.0.4"
   },
   "devDependencies": {
+    "@serverless/test": "^10.0.1",
+    "aws-sdk": "^2.1081.0",
+    "chai": "^4.3.6",
+    "chai-as-promised": "^7.1.1",
+    "fs-extra": "^10.0.1",
     "husky": "^7.0.4",
     "lint-staged": "^12.1.2",
+    "log": "^6.3.1",
+    "memoizee": "^0.4.15",
+    "mocha": "^9.2.1",
+    "node-fetch": "^2.6.7",
     "prettier": "^1.19.1",
-    "serverless": "^2.68"
+    "serverless": "^2.68",
```

Comment:
I was trying to avoid changes that seemed outside the scope of this PR, but maybe that's silly considering the new test utilities only really work with serverless 3. So let me know if you want me to also update serverless in this PR, it should be trivial since I've already been testing with that version